software/messaging/src/main/java/brooklyn/entity/messaging/rabbit/RabbitSshDriver.java

```diff
@@ -124,11 +123,20 @@ public void install() {
     @Override
     public void customize() {
         Networking.checkPortsValid(MutableMap.of("amqpPort", getAmqpPort()));
-        newScript(CUSTOMIZING)
-                .body.append(
-                    format("cp -R %s/* .", getExpandedInstallDir())
-                )
-                .execute();
+        copyTemplate(entity.getConfig(RabbitBroker.CONFIG_TEMPLATE_URL), getConfigPath() + ".config");
+        ScriptHelper scriptHelper = newScript(CUSTOMIZING);
+
+        scriptHelper.body.append(
+                format("cp -R %s/* .", getExpandedInstallDir())
+        );
+
+        if (entity.getConfig(RabbitBroker.ENABLE_MANAGEMENT_PLUGIN)) {
+            scriptHelper.body.append(
+                    "./sbin/rabbitmq-plugins enable rabbitmq_management"
+            );
+        }
+
+        scriptHelper.execute();
```

Comment:
Appreciate there wasn't one before but would make sense to include a `failOnNonZeroResultCode` to this step.